cypress/fixtures/pages/liveblogs.ts

```diff
@@ -7,8 +7,8 @@ const liveblogs: Page[] = [
 	{
 		path: getTestUrl(
 			stage,
-			'/politics/live/2022/jan/31/uk-politics-live-omicron-nhs-workers-coronavirus-vaccines-no-10-sue-gray-report?live=true',
-			{ isDcr: true },
+			'/politics/live/2022/jan/31/uk-politics-live-omicron-nhs-workers-coronavirus-vaccines-no-10-sue-gray-report',
```

Comment:
Removing `?live=true` causes the other Cypress-3 test to fail. I am fixing the other test in my PR, which will fix this one too, so happy for you to leave it

Reply:
It's still there just added here now: https://github.com/guardian/commercial/pull/992/files#diff-188823ef01a17b9942962471319b2f75645e1bea6bd18c38eafb078ae0daf56aR29

Reply:
Ah the urls weren't quite correct for liveblogs when running locally, fixed.